### ISSUE TITLE
[0.106.0b0] Uncaught TypeError: Cannot assign to read only property 'index' of object

### ISSUE THREAD

**Issue by TheZoker** [kalkih/mini-graph-card#274]
Since the new beta 0.106.x I get this error in my frontend:
```
mini-graph-card-bundle.js:1 Uncaught TypeError: Cannot assign to read only property 'index' of object '#<Object>'
    at mini-graph-card-bundle.js:1
    at Array.forEach (<anonymous>)
    at HTMLElement.set hass [as hass] (mini-graph-card-bundle.js:1)
    at HTMLElement.value (hui-view.ts:61)
    at HTMLElement.value (hui-view.ts:337)
    at HTMLElement.<anonymous> (hui-view.ts:68)
    at r (fire_event.ts:76)
    at create-element-base.ts:72 
```
Also lovelace does not display anything anymore. 
When I remove mini graph card js, the error is gone and lovelace loads normal again.

**Comment by cogneato** [comment]
Noticed this too late after submitting my own duplicate 106.0b0 issue (now closed).

**Comment by kalkih** [comment]
Thanks, will have a look at the changed this weekend.

**Comment by DavidFW1960** [comment]
Damn I was hoping I could merge the pr manually to get it working again.....

**Comment by kalkih** [comment]
@DavidFW1960 Clone this repo, checkout the 0.106.x-compatible branch and build from source as per developer instructions in the readme 😉

**Comment by DavidFW1960** [comment]
I'm sure that's simpler than it looks but above my pay grade I think

**Comment by DavidFW1960** [comment]
I'm clearly cleverer than I thought I am as I managed to build it and it seems to be working...

**Comment by DavidFW1960** [comment]
OK it works but it causes the whole GUI for any view with that card on it to slow down dramatically... to the point of being unusable.

**Comment by TheZoker** [comment]
@DavidFW1960 Do you use the history card?
Could be related to this error:
https://github.com/home-assistant/home-assistant-polymer/pull/4935

**Comment by DavidFW1960** [comment]
@TheZoker 100% correct.. it's the history card slowing everything down. I rolled back to 0.105.5 nd @kalkih custom-card I built as per above works fine in 0.105.5 and 0.106.x as well. Just waiting for b1 now.

**Comment by amundmr** [comment]
I am still having these problems. Using the newest provided .js file does not work. I also tried building my own from the branch for 106.x without success. I just upgraded to 106.6. Any idea?
`2020-03-15 15:54:39 ERROR (MainThread) [frontend.js.latest.202002205] http://ip/local/community/mini-graph-card/mini-graph-card-bundle.js?v=0.9.3:1:75221 Uncaught TypeError: Cannot assign to read only property 'index' of object '#<Object>'`

Best regards
